plugins/modules/keycloak_client.py

```diff
@@ -108,13 +108,14 @@
 
     client_authenticator_type:
         description:
-            - How do clients authenticate with the auth server? Either V(client-secret) or
-              V(client-jwt) can be chosen. When using V(client-secret), the module parameter
-              O(secret) can set it, while for V(client-jwt), you can use the keys C(use.jwks.url),
+            - How do clients authenticate with the auth server? Either V(client-secret),
+              V(client-jwt) or V(client-x509) can be chosen. When using V(client-secret), the module parameter
```

Comment:
```suggestion
              V(client-jwt), or V(client-x509) can be chosen. When using V(client-secret), the module parameter
```
https://docs.ansible.com/ansible/devel/dev_guide/style_guide/grammar_punctuation.html#commas